ansible_runner/utils.py

```diff
@@ -82,6 +83,25 @@ def check_isolation_executable_installed(isolation_executable):
         return False
 
 
+def stream_dir(directory, **additional):
```

Comment:
this doesn't seem to work with non-absolute paths for `directory` like `~/Documents/repos/openshift`. I'm unclear why.

Reply:
Presumably because `~/` is a shell-evaluated alias for `$HOME`, or something like that.